api/scpca_portal/templates/readme/terms_of_use/additional_restrictions_table.md

```diff
@@ -0,0 +1,6 @@
+### Additional Restrictions
+| Project ID | Data Usage Restrictions |
+| :--------- | :---------------------- |
+{% for project in projects %}
+| [{{ project.scpca_id }}]({{ project.url }}) | {{ project.additional_restrictions|default:"No additional restrictions" }} |
```

Comment:
I don't think this will work how we intend it to:
> If value is "" (the empty string), the output will be nothing.
https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#default

Reply:
I think you need to use `firstof` here

Reply:
The doc indicates that if there is such a case below:
> ```
> {{ value|default:"nothing" }} 
> ```
> If value is "" (the empty string), the output will be **`nothing`** 

So it provides support for empty strings. The choice of this word in the doc is a bit confusing 😅!

(Btw, thank you for introducing me to the `firstof` filter which is really useful 👍)